.github/workflows/release.yml

```
@@ -0,0 +1,286 @@
+          # Verify installation
+          ./change-site.sh --version
+
+          # Test with dry-run
+          ./change-site.sh --dry-run 192.168 172.23
+          \`\`\`
+          EOF
+
+          echo "Release notes generated for $version"
+
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)